doc/sphinx/administration_portal/client/vpbx/routing_tools/match_lists.rst

```diff
@@ -27,4 +27,7 @@ The section **Client configuration** > **Match Lists** allows to configure
 different items that will group the numbers and patterns.
 
 As shown in **List of Match List Patterns**, a match list can contain specific numbers or groups using
-`Regular Expressions <http://php.net/manual/en/reference.pcre.pattern.syntax.php>`_
\ No newline at end of file
+`Regular Expressions <http://php.net/manual/en/reference.pcre.pattern.syntax.php>`_
+
+.. error:: **Regular expressions** of Match List patterns must be **in E.164** format and **plus symbol must be
+               escaped** (e.g. ^\\+34 for all spanish numbers).
```

Comment:
`^\+34` is the correct format for the spanish regular expression

Reply:
Two slashes are needed to show just one ;)